gen-api-reference/README.md

```diff
@@ -0,0 +1,34 @@
+This folder contains scripts that automatically generate API reference pages for Kubeflow.
+The tool used here can be found at https://github.com/ahmetb/gen-crd-api-reference-docs.
```

Comment:
It's not clear whether this directory will contain various API reference generation tools, or just the tool for CRD reference. I think the directory will contain various, but this page currently only describes CRD because that's the only one we have. If that's a correct assumption, then I think you should add a heading between line 1 and line 2, something like this:

```
## Reference docs for Kubernetes custom resources
```